mack/__init__.py

```diff
@@ -311,7 +425,23 @@ def append_without_duplicates(
     ).whenNotMatchedInsertAll().execute()
 
 
-def is_composite_key_candidate(delta_table: DeltaTable, cols: List[str]) -> bool:
+def is_composite_key_candidate(delta_table: DeltaTable, cols: List) -> bool:
```

Comment:
@joaopfmsantos why did you remove the `[str]` hint here?

Reply:
Apologies, you are correct :) Reverted